modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -175,6 +177,31 @@ Method Label Options Color:: Specifies the OPTIONS method label color.
 Method Label Head Background Color:: Specifies the HEAD method label background color.
 Method Label Head Color:: Specifies the HEAD method label color.
 
+If an *API Console* navigation item such as an API endpoint or method has a long name, it wraps around to a second line. By default, the line breaks at the end of a word. To configure the line break behavior:
+
+. In the API Community Manager control panel, click Community Builder.
```

Comment:
What is the user clicking: something LABELLED *Community Builder*?

Reply:
Yes, this is a label. Fixed formatting.